recipe/build.sh

```diff
@@ -115,10 +119,10 @@ cd build
   --disable-bootstrap \
   --disable-multilib \
   --enable-long-long \
-  --enable-default-pie \
   --with-sysroot=${PREFIX}/${TARGET}/sysroot \
   --with-build-sysroot=${BUILD_PREFIX}/${TARGET}/sysroot \
   --with-gxx-include-dir="${PREFIX}/${TARGET}/include/c++/${gcc_version}" \
-  $GCC_CONFIGURE_OPTIONS
+  --with-pkgversion="conda-forge ${gcc_version}-${PKG_BUILDNUM}" \
+  --with-bugurl="https://github.com/conda-forge/ctng-compilers-feedstock/issues/new/choose"
```

Comment:
Can you only do this if `${channel_targets} == conda-forge*`?

Reply:
addressed in https://github.com/conda-forge/ctng-compilers-feedstock/pull/91/commits/676e1c520cf18b6f058bc12485ce13cb1fdc3146